### DISCUSSION TITLE
Smooth scrolling

### DISCUSSION THREAD

**Discussion by KillianLobato** [sumatrapdfreader/sumatrapdf#3218] [Ideas]
Hi everyone,
I started using sumatra today. Really impressed with this software. Well done! I am a big fan of smooth scrolling of windows. I have even purchased software to specifically address this issue in windows - big call out to SmoothScroll! You can tune the smoothness of scrolling on a per-app basis. Amazing!
Anyway, after installing Sumatra and playing with the advanced settings, I see there was an option for smooth scrolling. However it is "basic", it is either on or off with no tuning. I have disabled it and am currently using SmoothScroll to get the best experience possible.

Also, I suggest that people have a look at Xodo PDF reader. The way it animates scrolling and zooming in and out in unbeatable. However, the SmoothScroll software refused to work for me on this, and Xodo is a bit slow to load.

Again, nice one, and lets get nice smooth scrolling working in windows programs!

Regards,
KL

**Comment by GitHubRulesOK** [comment]
SumatraPDF scroll speed / steps are related to feedback by the system you don't always need extra software to get smooth scroll if you can adjust the OS and device driver input settings. This is similar to comments about printing, where SumatraPDF uses the default output settings that Windows, and its print drivers offer.

The best way to get smooth scroll is use a device with auto scroll where the steps are as small as a scan line rather than a number of scan lines at a time. So I can scroll damn slow say at 10 lines per minute no problem. or roll 6 times per line if windows permits it.
